distributed/tests/test_scheduler.py

```diff
@@ -3595,3 +3595,11 @@ async def test_worker_state_unique_regardless_of_address(s, w):
     assert ws1 is not ws2
     assert ws1 != ws2
     assert hash(ws1) != ws2
+
+
+@gen_cluster(nthreads=[("", 1)])
+async def test_scheduler_close_fast_deprecated(s, w):
+    ws1 = s.workers[w.address]
+    host, port = parse_host_port(ws1.address)
```

Comment:
Minor comment: Do we need these two lines? I don't see `ws1`, `host`, or `port` being used